### ISSUE TITLE
TBomb Calling is not working!

### ISSUE THREAD

**Issue by ajstars1** [TheSpeedX/TBomb#361]
**Describe the bug**
Call bombing is not working from a year later. Need to be update
I guess it need to be update its call API's.

**Comment by william123H** [comment]
call bombing is not working

**Comment by sudu0** [comment]
Yes call bombing is not works it shows that the calls are being made but they aren't and I am from India so what's the problem is it something that is related to country problem.

**Comment by 0n1cOn3** [comment]
Please read the README exactly. It's explained why it doesn't work properly!

**Comment by 0n1cOn3** [comment]
> **Describe the bug**
> Call bombing is not working from a year later. Need to be update
> I guess it need to be update its call API's.

Thats correct. 
but: TBomb was made for SpeedX's own education. So its unknown when the APIs gonna be updated as long someone else made a PR.

**Comment by TheSpeedX** [comment]
Hello!

After careful consideration, I’ve decided to close all open issues for this repository. 
This project has grown into something beyond my initial expectations, but I’ve found it increasingly difficult to maintain due to low-quality contributions and spam pull requests/issues. Furthermore, I am transitioning to new ventures, and I won’t be able to continue contributing to TBomb any further.

Though I’m stepping away, I’m grateful to all those who have used and supported this tool. TBomb will remain usable through Git and PyPI, but I won’t be actively involved moving forward.

Thank you for your understanding and for using this tool!
Best regards,
TheSpeedX
